qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -1,5 +1,5 @@
 {
-  "gaTrackingId": "UA-119127212-1",
```

Comment:
anyway we could use clearer IDs everywhere? include `qa-covid19` in the name instead of a random number? is `119127212` random or some kind of account number?

Reply:
nope, Google manage that @paulineribeyre

Reply:
I think that number is teh GA account no

Reply:
too bad

Reply:
GA account no plus the auto increment number of the ID under that account @mfshao @paulineribeyre